stdlib/asm/collections/smt.masm

```diff
@@ -403,6 +487,11 @@ export.get
         adv.push_mapvaln adv_push.1
         # => [leaf_size, NV, K, R]
 
+        # Consistency check: assert this is actually a valid leaf.
+        # (18 cycles).
+        dup u32lte.MAX_LEAF_SIZE u32assert.err="invalid leaf: larger than maximum size of 8192" drop
```

Comment:
I'm not sure this will work as intended. We can't use `u32lte` unless we know that `leaf_size` is a `u32`. So, I'd write this as:
```
dup u32assert2 u32lte.MAX_LEAF_SIZE assert.err="invalid leaf: larger than maximum size of 8192"
```
This should be 9 cycles.